tasks/css-tools.js

```diff
@@ -0,0 +1,51 @@
+/*
```

Comment:
These are no longer being called in this branch I believe so can we remove them from this PR?

Reply:
Yes Sure